proto/zetachain/zetacore/observer/query.proto

```diff
@@ -150,6 +151,30 @@ service Query {
       returns (QueryAllChainNoncesResponse) {
     option (google.api.http).get = "/zeta-chain/observer/chainNonces";
   }
+  // Queries the TssFundMigratorInfo for a specific chain
+  rpc TssFundsMigratorInfo(QueryTssFundsMigratorInfoRequest)
+      returns (QueryTssFundsMigratorInfoResponse) {
+    option (google.api.http).get = "/zeta-chain/observer/getTssFundsMigrator";
+  }
+
+  // Queries all TssFundMigratorInfo
+  rpc TssFundsMigratorInfoAll(QueryTssFundsMigratorInfoAllRequest)
+      returns (QueryTssFundsMigratorInfoAllResponse) {
+    option (google.api.http).get =
+        "/zeta-chain/observer/getAllTssFundsMigrators";
+  }
+}
+
+message QueryTssFundsMigratorInfoAllRequest {}
+
+message QueryTssFundsMigratorInfoAllResponse {
+  repeated TssFundMigratorInfo tss_funds_migrators = 1
+      [ (gogoproto.nullable) = false ];
+}
+
+message QueryTssFundsMigratorInfoRequest { int64 chain_id = 1; }
+message QueryTssFundsMigratorInfoResponse {
+  TssFundMigratorInfo tss_funds_migrator = 1 [ (gogoproto.nullable) = false ];
```

Comment:
We need to update camel case for these as well

Reply:
I can change it but we have been using this format fields in a struct before, for example 

https://github.com/zeta-chain/zeta-node/blob/3ce96875387aecd2302b74b2d6f9d2bce8e30e26/proto/zetachain/zetacore/observer/query.proto#L206-L208

https://github.com/zeta-chain/zeta-node/blob/3ce96875387aecd2302b74b2d6f9d2bce8e30e26/proto/zetachain/zetacore/observer/query.proto#L221-L224